src/main/java/com/mraof/minestuck/data/recipe/MinestuckGristCostsProvider.java

```diff
@@ -184,6 +190,13 @@ protected void buildCraftingRecipes(Consumer<FinishedRecipe> recipeSaver)
 		GristCostRecipeBuilder.of(Items.POISONOUS_POTATO).grist(AMBER, 4).grist(IODINE, 2).buildFor(recipeSaver, Minestuck.MOD_ID);
 		GristCostRecipeBuilder.of(Items.MELON_SLICE).grist(AMBER, 1).grist(CAULK, 1).buildFor(recipeSaver, Minestuck.MOD_ID);
 		GristCostRecipeBuilder.of(Items.ENCHANTED_GOLDEN_APPLE).grist(AMBER, 4).grist(GOLD, 150).grist(URANIUM, 10).buildFor(recipeSaver, Minestuck.MOD_ID);
+		GristCostRecipeBuilder.of(Items.BIG_DRIPLEAF).grist(URANIUM, 2).grist(SHALE, 2).grist(MERCURY, 4).buildFor(recipeSaver, Minestuck.MOD_ID);
+		GristCostRecipeBuilder.of(Items.SMALL_DRIPLEAF).grist(URANIUM, 1).grist(SHALE, 1).grist(MERCURY, 2).buildFor(recipeSaver, Minestuck.MOD_ID);
+		GristCostRecipeBuilder.of(Items.HANGING_ROOTS).grist(MERCURY, 10).grist(BUILD, 5).buildFor(recipeSaver, Minestuck.MOD_ID);
+		GristCostRecipeBuilder.of(Items.MOSS_BLOCK).grist(URANIUM, 6).buildFor(recipeSaver, Minestuck.MOD_ID);
```

Comment:
I strongly recommend a different grist type be used for moss or at least a lower quantity. Due to its ability to rapidly create more of itself on contact with stone, players could abuse this as a quick source of uranium. When converted to a universal cost(equivalent to build grist), normal stone blocks are valued at 2 and moss would be valued at 30